README.md

```diff
@@ -1,3 +1,12 @@
 # Let's start cross stich
 
-TBA
+This repository contains chapters on cross stitching, primarily written by Dr. Sarah Gibson.
+
+[_The Turing Way_](https://github.com/alan-turing-institute/the-turing-way) team members wanted to test The Carpentries lesson Template so that they could use it for hosting lessons relevant for The Turing Way project.
```

Comment:
```suggestion
[_The Turing Way_](https://github.com/alan-turing-institute/the-turing-way) team members wanted to test The Carpentries [lesson template](https://github.com/carpentries/lesson-example) so that they could use it for hosting lessons relevant for _The Turing Way_ project.
```